### PR TITLE
Improved Python Authentication Samples

### PR DESCRIPTION
**Pull request by berndverst**

### CODE REVIEW

**Comment by berndverst** [comment]
cc @lmazuel

File: docs-ref-conceptual/python-sdk-azure-authenticate.md [hunk 1]
```diff
@@ -23,36 +23,53 @@ Several options are available to authenticate your application with Azure when u
...
+Next, create a client object to start working with the API:
 
 ```python
 from azure.mgmt.compute import ComputeManagementClient
 
-# change for your actual subscription id
+# Your Azure Subscription ID
 subscription_id = '33333333-3333-3333-3333-333333333333'
 
 client = ComputeManagementClient(credentials, subscription_id)
```
Comment (lmazuel) [inline docs-ref-conceptual/python-sdk-azure-authenticate.md]: Add a note, that to connect to sovereign clone, base_url on the client should be updated to the RESOURCE value used earlier.

Comment (berndverst) [inline docs-ref-conceptual/python-sdk-azure-authenticate.md]: Done.

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/python-sdk-azure-authenticate.md](https://github.com/berndverst/azure-docs-sdk-python/blob/aug-1-stage/docs-ref-conceptual/python-sdk-azure-authenticate.md) | :white_check_mark:Succeeded | |
[docs-ref-services/postgresql.md](https://github.com/berndverst/azure-docs-sdk-python/blob/aug-1-stage/docs-ref-services/postgresql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C19%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201707191723545367-73%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**